<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTraineeDataRecorderModule</class>
 <widget class="qSlicerWidget" name="qSlicerTraineeDataRecorderModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>698</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Display</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="BufferGroupBox">
        <property name="title">
         <string>Buffer</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="ControlsGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Controls</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="StatisticsGroupBox">
        <property name="title">
         <string>Statistics</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="NumTransformsLabel">
           <property name="text">
            <string>Number of Transforms:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="TotalTimeLabel">
           <property name="text">
            <string>Total Time (s):</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="TotalTimeResultLabel">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="NumTransformsResultLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="NumMessagesLabel">
           <property name="text">
            <string>Number of Messages:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="NumMessagesResultLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="MessagesGroupBox">
        <property name="title">
         <string>Messages</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="ctkCollapsibleButton" name="HelloCollapsibleButton">
     <property name="text">
      <string>Trainee Data Recorder</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <widget class="QGroupBox" name="groupBoxAuthentication">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>311</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Authentication</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditUsername_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditPassword">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonLogin">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>110</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonCreate">
       <property name="geometry">
        <rect>
         <x>194</x>
         <y>140</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Create New User</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxTraineeData">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Trainee Data</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Student Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditName">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>130</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Save </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Student ID:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditStudentID">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="MessageLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>To enter trainee data, please login or create new user.</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
